modules/data-loading/pages/data-loading-ha.adoc

```diff
@@ -0,0 +1,25 @@
+= Data Loading High Availability (HA)
+:page-aliases:
+
+Data Loading High Availability (HA) provides load balancing and automatic fail over in the event of a service disruption.
+
+== `Setup`
+First, set up a cluster with replication factor greater than 1 following xref:cluster-and-ha-management:ha-cluster.adoc[High Availability Cluster Configuration]
+
+Second, enable Kafka Connect V2 using `gadmin` commands.
+[source, bash]
+----
+gadmin config set KafkaConnect.EnableV2 true
+----
+
+Then run config apply and restart dependency services.
```

Comment:
Let's show the specific command.
And we can use the new config apply --restart-deps option

Reply:
Yes. The command is right below